components/experimental/src/dimension/provider/extended_currency.rs

```diff
@@ -43,5 +43,5 @@ pub struct CurrencyExtendedDataV1<'data> {
     ///    Regards to the [Unicode Report TR35](https://unicode.org/reports/tr35/tr35-numbers.html#Currencies),
     ///    If no matching for specific count, the `other` count will be used.
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub display_names: PluralElementsV1<'data>,
+    pub display_names: PluralElementsPackedCowStr<'data>,
```

Comment:
We don't typically call things `V1` unless they are actually data structs. I let it land with this name initially because it's unstable, but I consider it an improvement to call the type by what it actually is.